File: 프로그래머스/level1/수박수박수박수박수박수.java
class Solution {
    public String solution(int n) {
        String answer = "";
        int twice = n/2;
        for(int i = 0;i < twice;i++)
            answer = answer + "수박";
        
        if(n%2 == 1)
            answer = answer + "수";
        
        return answer;
    }
}

File: programmers/level1/수박수박수박수박수박수.cpp
#include <iostream>
#include <string>
#include <vector>
#include <cstring>

using namespace std;

string solution(int n) {
    string answer = "";
    int num = n / 2;
    int i;
    
    for(i = 0;i < num;i++)
        answer += "수박";
        
    if(n % 2 == 1)
        answer += "수";
       
    cout << answer;
    
    return answer;
}